src/storage/influx.js

```diff
@@ -218,11 +218,11 @@ class InfluxStorage {
 
       if (ts > pointTs) {
         for (let exchange in point) {
-          if (!point.close) {
-            point.low = null
-            point.high = null
-            point.open = null
-            point.close = null
+          if (!point[exchange].close) {
```

Comment:
Im not sure this fix the issue tbh, we do this don't we lose the reference to previous candle price ?
candle 1 close at 6954
candle 2 only receive liquidations, close is null you set all ohlc to null
candle 3 new price (no reference since previous candle close is null) => might create a gap

Reply:
I'm confused by this logic, can you explain please why we set ohlc to null here at all?

I see this: on each trade we check it the trade's timestamp is bigger than previous. If yes - we try to nullify all values from all exchanges (Why?), save it as a refPoint, then push this exchange to points array (Why?). Then if there's no exchange point we check the refPoint or create a new one. And then we fill it with values.

I don't get why we would do that part of nullifying all the exchanges, although it seems like doesn't work now because of typo.